notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>A quick description related to this function like this: </p><blockquote>A function to extract and convert the event time to datetime object. If the initial extraction fails, it will convert the entire DataFrame to a string and extract the datetime in a different format.</blockquote><p> </p><p>(If I missunderstood the function, please feel free to correct it) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p><strong>This function looks like it not being used.</strong> Let me know if somethings goes wrong, when it deletes.</p>